lib/logstash/devutils/rspec/spec_helper.rb

```diff
@@ -17,6 +17,10 @@
 require "logstash/devutils/rspec/logstash_helpers"
 require "insist"
 
+unless LogStash::Environment.const_defined?(:LOGSTASH_HOME)
+  LogStash::Environment::LOGSTASH_HOME = File.expand_path("../../../../../", __FILE__)
```

Comment:
yeah, agree,  __FILE__ will not necessarily point us to the right place. in this case I think I would set it to `Dir.pwd` so it will point to the dir where you launched tests...